src/Messages/index.js

```diff
@@ -4,8 +4,12 @@ export default {
     login: 'login',
     schedule: 'Plan',
     loading: 'Ładowanie',
+    edit: 'Edytuj',
     save: 'Zapisz',
     passwordTooShort: 'hasło jest za krótkie!',
+    profileHintPosition: 'Stanowisko',
```

Comment:
idea tego pliku to słowo ang : słowo pol, wtedy np. "Nazwa firmy" jest reużywalne + jesli prawą stronę przetłumaczymy, to dostaniemy nowy język ;) idąc tym tropem po polu profileHintPosition spodziewałbym się stringa Profil Sugestia Pozycja :P